### PR TITLE
Functions for getting pubmed ids and rsids

### PR DESCRIPTION
**Pull request by hsiaoyi0504**

### CODE REVIEW

**Comment by hsiaoyi0504** [comment]
The remaining part is `get_pubmed_ids_from_phenotypes`.

File: dc/utils.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import six.moves.urllib
+from Bio import Entrez, Medline
+import re
+
+Entrez.email = "trivneel211@gmail.com"
+
+
+def get_pubmed_ids_from_phenotypes(phenotypes):
+    pubmed_ids = []
+    # TODO
```
Comment (nathan-wan) [inline dc/utils.py]: didn't we finish this already?  @NeelayTrivedi

**Comment by NeelayTrivedi** [comment]
Yeah .

File: dc/utils.py (hunk 1 above)
Comment (hsiaoyi0504) [reply dc/utils.py]: I think we have a prototype in https://github.com/NCBI-Hackathons/DiseaseClusters/blob/a53b748e79a953334d88c6eab1771a59011d4d3d/get_abstracts.py#L87-L98

Comment (hsiaoyi0504) [reply dc/utils.py]: However, I think the conclusion we have yesterday is to use PubMed search directly.

**Comment by hsiaoyi0504** [comment]
@nathan-wan @NeelayTrivedi Can you review again ?

**Pull request merged**